react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+const colorChange = {
+  3: 'purple',
+  6: 'blue',
+  9: 'red',
+  12: 'orange',
+  15: 'yellow',
+  18: 'white'
+};
+
+let thisManyClicks = 0;
+
+class HotButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { clickCount: thisManyClicks };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event) {
+    thisManyClicks++;
+    this.setState({ clickCount: thisManyClicks });
+    if (thisManyClicks.toString() in colorChange) {
+      event.target.style.backgroundColor = colorChange[thisManyClicks.toString()];
+    }
```

Comment:
The problem with this approach is that it doesn't actually use the state change to update what the user sees. Try adding conditional logic to your render method that chooses the correct CSS class name instead of mutating the DOM directly.